third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -657,7 +657,7 @@ func TestAccComputeInstance_scratchDisk(t *testing.T) {
 				Config: testAccComputeInstance_scratchDisk(instanceName),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckComputeInstanceExists(
-						t, "google_compute_instance.scratch", &instance),
```

Comment:
Why was this change made?

Reply:
This one was odd, not sure how/why it's just failing now, but `computeInstanceImportStep` specifically imports resource name `"google_compute_instance.foobar"` so this test was failing.